kbs/src/admin/password.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2025 by NVIDIA.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use actix_web::{http::header::Header, HttpRequest};
+use actix_web_httpauth::headers::authorization::{Authorization, Bearer};
+use argon2::{Argon2, PasswordHash, PasswordVerifier};
+use jwt_simple::{
+    claims::NoCustomClaims,
+    common::VerificationOptions,
+    prelude::{Claims, Ed25519KeyPair, EdDSAKeyPairLike, EdDSAPublicKeyLike},
+};
+use log::info;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+use crate::admin::error::*;
+use crate::admin::AdminBackend;
+
+const ADMIN_TOKEN_ISSUER: &str = "Trustee Admin";
+
+#[derive(Clone, Debug, Default, Deserialize, PartialEq)]
+#[serde(default)]
+pub struct PasswordAdminConfig {
+    pub personas: Vec<PasswordPersona>,
+    /// The number of hours that an admin token will be valid for.
+    /// After the token expires, an admin must login again and receive
+    /// a new token.
+    pub admin_token_life_hours: u64,
```

Comment:
Let's set this to seconds than hours for a better grained.

Reply:
Seconds seems too small. How about minutes? I am thinking that people probably won't want to login in very often; maybe every day or two.

Reply:
Yes seconds are small. What matters hare is grain. No matter minutes or seconds the user can configure a big number. I am ok with minutes.